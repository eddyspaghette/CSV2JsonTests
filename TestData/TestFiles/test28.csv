ッ!
ド!え!さ!℀!ↂ!🍕
ツ!!ぎ!せ!℟!▓
ト!ぇ!ご!₣!ψ!😵
ヅ!い!け!₠!Å!○
ミ!ぅ!げ!₡!ℬ!☝
デ!う!こ!₢!!⭐
