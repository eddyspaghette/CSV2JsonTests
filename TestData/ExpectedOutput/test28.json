[
 {
   "ッ!": "ド!え!さ!℀!ↂ!🍕"
 },
 {
   "ッ!": "ツ!!ぎ!せ!℟!▓"
 },
 {
   "ッ!": "ト!ぇ!ご!₣!ψ!😵"
 },
 {
   "ッ!": "ヅ!い!け!₠!Å!○"
 },
 {
   "ッ!": "ミ!ぅ!げ!₡!ℬ!☝"
 },
 {
   "ッ!": "デ!う!こ!₢!!⭐"
 }
]